python/python/tests/test_transaction.py

```diff
@@ -282,6 +283,19 @@ async def transaction_test(
     args.append("0-2")
     transaction.xtrim(key11, TrimByMinId(threshold="0-2", exact=True))
     args.append(1)
+
+    min_version = "7.0.0"
+    if not await check_if_server_version_lt(redis_client, min_version):
+        # TODO: change it to pytest fixture after we'll implement a sync client
+        zset_key = "{{{}}}:{}".format(keyslot, get_random_string(3))
```

Comment:
why is it not not next to the rest of the keys?

Reply:
No particular reason, just because this is in a separate block I guess. I'll move it to be with the rest of the keys